package.json

```diff
@@ -9,12 +9,14 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.48",
     "@reduxjs/toolkit": "^1.3.4",
+    "ci": "^1.0.0",
     "express": "^4.17.1",
```

Comment:
Also, why do we have express? :/

Reply:
it appears Tuan added that when we first started the repo! that has been there for a while. I dont think we need it, but ill test that before removing it

Reply:
ehm, we need that for heroku

Reply:
please dont remove express